### PR TITLE
Add Forms

### PR DESCRIPTION
**Pull request by niicode**
Description
In this project I achieved the following:

Create a method called current_user in your ApplicationController to make current user data available to all controllers.
It will return the first user from the database.
Don't worry about proper authentication at this time. You will add it to the upcoming projects.
Create forms to perform the following functions:
Create a Post on behalf of the current_user (use the method you created in your ApplicationController).
Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).
Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    @comment = current_user.comments.new(comment_params)
+    @comment.post_id = params[:post_id]
+
+    if @comment.save
+      redirect_to user_post_path(current_user, @comment.post)
+    else
+      render :create
+    end
+  end
+end
```
Comment (codecaiine) [inline app/controllers/comments_controller.rb:25]: - [ ] [OPTIONAL] I think it should be great to have comment params method.  Kindly follow this link [about the strong parameters](`https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides`) :smile:

File: config/routes.rb [hunk 2]
```diff
@@ -2,5 +2,7 @@
   # Rails.application.routes.draw do
   resources :users, only: %i[index show] do
     resources :posts, only: %i[index show]
+    resources :comments, only: [:create]
+    resources :likes, only: [:create]
```
Comment (codecaiine) [inline config/routes.rb]: - [ ] Good job so far but unfortunately I can't find  GET method  `posts#new ` to get a post and POST method `posts#create ` to create a new post and also `comments#new`. Please fix it to make your app works as expected :+1:

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (codecaiine) [inline app/controllers/comments_controller.rb:25]: - [ ] I think there is an issue in your schema when I tried to create for example a new user some attributes are missing in the Database. I would like strongly recommend working on your database and adding new migrations to update to let the user be able to create a new user and a new post and so on. please check this [article](https://stackoverflow.com/questions/1992019/how-can-i-rename-a-database-column-in-a-ruby-on-rails-migration). Please fix to make your app work as expected :+1:

Comment (codecaiine) [inline app/controllers/comments_controller.rb:25]: - [ ] As I mentioned above Your project run well and but I am not able to create a user from the rails console and I can't see the forms created which is required for this project :cry: Please fix your database and migration and also work on the different method that makes you app work as expected and make sure to populate your database in order to see the form created and your project fit the given [wireframes](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES). Also make sure that we are able to comment a post and also like it :+1:

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @niicode  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good project setup :heavy_check_mark: 
- No rubocop errors :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- **I would like to strongly recommend you make your commit messages more meaningful in the future.** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @niicode , 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**